home.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>HELLO WORLD</h1>
```

Comment:
This doesn't look right, have you thought about maybe trying this?

```suggestion
    <h1>HELLO  NEW WORLD</h1>
```

This might be more readable